tools/tasks/build.assets.dev.ts

```diff
@@ -1,13 +1,12 @@
 import {join} from 'path';
 import {PATH} from '../workflow.config';
 
-export = function (gulp) {
+export = function task(gulp) {
```

Comment:
Calling the function based on what it does seems more appropriate semantically, i.e. instead of calling the task `task`, you can name it `buildAssets` or something like that. This way if we have a stack trace somewhere we'll get better information about what actually have happened.